src/table.ts

```diff
@@ -683,16 +683,20 @@ Please use the format 'prezzy' or '${instance.name}/tables/prezzy'.`);
       rowsLimit = options.limit;
     }
 
-    const userStream = through.obj();
+    const userStream = through.obj((rowData, enc, next) => {
```

Comment:
I hate asking this question, but Imma do it anyway.  What would happen if you replaced this line with:

```js
const userStream = new stream.PassThrough({objectMode: true});
```

Theoretically ...  these are doing the exact same thing.

Reply:
Sure. Ben and I hopped on a call and this came up a few times.

Reply:
So, they are the same thing as far as I can tell. We have two other uses of through in this file. Though we may be able change those out for Readable